draft-ietf-rats-eat.md

```diff
@@ -236,10 +236,44 @@ The notion of attestation here is large and may include, but is not limited to t
 This document uses the terminology and main operational model defined in [RATS.architecture].
 In particular it is a format that can be used for Attestation Evidence or Attestation Results as defined in the RATS architecture.
 
+## Entity Overview
+
+The document uses the term "entity" to refer to the target of the attestation token.
+The claims defined in this document are claims about an entity.
+
+An entity is an implementation in hardware, software or both.
+
+An entity roughly corresponds to a "system component" as defined in the Internet Security Glossary {{RFC4949}}.
```

Comment:
```suggestion
An entity corresponds to a "system component" as defined in the Internet Security Glossary {{RFC4949}}.
```

Reply:
If that feels like it is not enough, please add text that specifically elaborates on the difference.